src/python/WMCore/WMBS/MySQL/Create.py

```diff
@@ -65,6 +65,26 @@ def __init__(self, logger = None, dbi = None, params = None):
           """CREATE UNIQUE INDEX uniq_wmbs_file_run_lumi on
              wmbs_file_runlumi_map (fileid, run, lumi)"""
 
+        #
+        # Insert LogCollect site to account for LogCollect output
+        #
+        self.inserts["99_location_logcollect_1"] = \
```

Comment:
Somewhere between 1.0.15.something and master this changed, we noticed this on the Tier0 side since we now have to add T2_CH_CERN to our resource control even though we never want to run jobs there. Just to be able to register the LogCollect output in WMBS.